exonum-java-binding/core/rust/exonum-java/src/main.rs

```diff
@@ -15,6 +15,7 @@
  */
 
 extern crate env_logger;
+extern crate exonum_btc_anchoring;
```

Comment:
Did just adding the anchoring trigger the problem? Or do we also have to add the service (= use the Service factory from anchoring)?